commands/itest_utils.go

```diff
@@ -131,6 +156,72 @@ func getLicensesEndpointURL(rtDetails *config.ArtifactoryDetails) string {
 	return fmt.Sprintf("%sapi/system/licenses", rtDetails.Url)
 }
 
+func setUpTargetRepositoryAndPermissions(ctx context.Context, t *testing.T, rtDetails *config.ArtifactoryDetails) {
+	createLogRepository(ctx, t, rtDetails)
+	setAnonAccess(ctx, t, rtDetails)
+	createAnonymousPermission(ctx, t, rtDetails)
+}
```

Comment:
Suggestion: rename the function to `enableAnonymousUpload` and maybe reorder calls inside:
1. `setAnonAccess`
2. `createLogRepository`
3. `createAnonymousPermission`